### ISSUE TITLE
Finetuning Avocodo vocoder degrades audio quality

### ISSUE THREAD

**Issue by tshmak** [DigitalPhonetics/IMS-Toucan#107]
Has anyone got decent results from finetuning the Avocodo vocoder and if so, how much audio is needed? 

My experience was that after finetuning for perhaps 600 epochs or so on 30mins of data, the vocoder was completely messed up. I'm using the release [v2.4](https://github.com/DigitalPhonetics/IMS-Toucan/tree/v2.4). 

The audio was in 22050Hz, but I don't suppose that causes any problems. 

I set `finetune=True`, and note that this resets the optimizer and the scheduler. Could this be the reason? 

I do note that the author mentioned [here](https://github.com/DigitalPhonetics/IMS-Toucan/issues/90#issuecomment-1460750869) that finetuning the vocoder is not recommended, although judging by the issues mentioning this, there's considerable interest in doing so in the community. 

Also, I suppose the standard training routine simply trains with the gold mel-spectrograms. Has anyone tried training with generated mel-spectrograms?

**Comment by tshmak** [comment]
Actually I tried again setting `finetune=False` and it seemed to work!
